chart/templates/controller.yaml

```diff
@@ -203,6 +200,67 @@ spec:
         {{- end }}
         securityContext:
           runAsUser: {{.Values.ControllerUID}}
+      - name: linkerd-proxy
```

Comment:
I would like to see these as a template as they're not being modified on a per-resource basis.

Reply:
@klingerf @siggy how does this interact with the CLI doing inject as well? Should these be behind an if when the CLI is rendering these files?

Reply:
Alright I added the proxy and init containers as sub-templates and then converted all the templates to `include` them instead of having them inline.